docs/source/prereqs.md

```diff
@@ -0,0 +1,231 @@
+# Prerequisites
+
+Let's start with Prerequisites:
+
```

Comment:
I would include a set of links to each platform section at the top here, similar to some of our tutorials.

Reply:
@nikhil550 I didn't see a specific one @nikhil550, but if you point me to one, I can keep them consistent.